### DISCUSSION TITLE
Tikz plots

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/precalculus#287] [General]
There are a number of ways to draw tikz plots, which we need to consider for LaTeX RAT files.

Here's one way:

```latex
\documentclass{standalone}
\usepackage{tikz}
\begin{document}
\begin{tikzpicture}[x=0.2in,y=0.2in]
\fill[white] (-11,-11) rectangle (11,11);
\draw[black!20,step=1] (-10,-10) grid (10,10);
\draw[thick] (0,-10) -- (0,10);
\draw[thick] (-10,0) -- (10,0);
\node[circle,fill] at (-4,0) {};
\node[anchor=north east] at (-4,0) {$A$};
\node[circle,fill] at (2,-6) {};
\node[anchor=north east] at (2,-6) {$B$};
\end{tikzpicture}
\end{document}
```

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/1559632/ecd5dbe5-b3b9-43b3-8e58-1601101015ee)

But there are others. Let's decide on the recommended style for TBIL PreCalc here.

**Comment by siwelwerd** [comment]
It puts the label a little farther away, but here is a one liner for labeled points:
```
\node[circle,fill,label={225:$C$}] at (3,3) {};
```

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/30658947/157499ec-6390-416a-a3d1-1dbda705d368)


The chief advantage being you only have to type the coordinates of the point once.  Note the 225 is the angle indicating placement of the label.

**Comment by tdegeorge** [comment]
@StevenClontz How do I add tick marks or values to the axes?

**Reply by siwelwerd** [reply]
Not sure of the best way, but here is a way

```
\documentclass{standalone}
\usepackage{tikz}
\begin{document}
\begin{tikzpicture}[x=0.2in,y=0.2in]
\fill[white] (-11,-11) rectangle (11,11);
\draw[black!20,step=1] (-10,-10) grid (10,10);
\draw[thick] (0,-10) -- (0,10);
\draw[thick] (-10,0) -- (10,0);
\node[circle,fill] at (-4,0) {};
\node[anchor=north east] at (-4,0) {$A$};
\node[circle,fill] at (2,-6) {};
\node[anchor=north east] at (2,-6) {$B$};
\node[circle,fill,label={225:$C$}] at (3,3) {};
\foreach \i in {-9,...,-1}
  {\node[anchor=north,thin,gray] at (\i,0) {$\i$};
  \node[anchor=north west,thin,gray] at (0,\i) {$\i$};}
\foreach \i in {1,...,9}
  {\node[anchor=north west,thin,gray] at (\i,0) {$\i$};
  \node[anchor=north west,thin,gray] at (0,\i) {$\i$};}
\end{tikzpicture}
\end{document}
```

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/30658947/fedd9141-8263-4a9c-a710-d7c44cc5f4ee)

**Reply by StevenClontz** [reply]
I think that'd need some adjustment so it's clear whether the numbers are for the line above or below (once you get far away from the center).

If these PDFs are just proofs of concept that will eventually be made in CheckIt [like this](https://teambasedinquirylearning.github.io/linear-algebra/2024e/exercises/#/bank/LE-ready/1/), then there's no need to over-engineer the Tikz code, as we'll need to produce the graphs in SageMath anyway...

**Reply by tdegeorge** [reply]
Ok, I updated the code in overleaf. It does look a little crowded around the A point...

https://www.overleaf.com/read/dzwsnptyphws#5fc1d9
